src/Germany/Mews.Fiscalizations.Germany.Tests/TransactionTests.cs

```diff
@@ -29,7 +29,7 @@ public async Task GetTransactionSucceeds()
             var retrievedStartedTransaction = await client.GetTransactionAsync(accessToken.SuccessResult, TestFixture.TssId, startedTransaction.SuccessResult.Id);
             Assert.AreEqual(retrievedStartedTransaction.SuccessResult.State, TransactionState.Active);
 
-            var finishedTransaction = await client.FinishTransactionAsync(accessToken.SuccessResult, TestFixture.ClientId, TestFixture.TssId, GetBill(), startedTransaction.SuccessResult.Id, lastRevision: "1");
+            var finishedTransaction = await client.FinishTransactionAsync(accessToken.SuccessResult, TestFixture.ClientId, TestFixture.TssId, GetBill(), startedTransaction.SuccessResult.Id, lastRevision: "2");
```

Comment:
What does this even mean? 😄

Reply:
The initial transaction must start with revision 1, and each update on the transaction must increment it by 1.